.github/workflows/release.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           path: /tmp/lce_android
           key: ${{ runner.os }}-${{ hashFiles('**/third_party/install_android.sh') }}
       - name: Configure Bazel
-        run: ./configure.sh <<< $'n\n'
+        run: python configure.py <<< ""
```

Comment:
Why do we need to explicitly pass "" when calling this script?

Reply:
It's interactive, so if you don't pass input it will prompt for input. It's possible that when you run it non-locally (i.e. not attached to a terminal) then it would work without this, I haven't actually tested it.

Reply:
I am pretty sure that GitHub actions will handle this correctly, so we should be able to safely change this to
```shell
./configure.py
```